openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx

```diff
@@ -428,12 +428,12 @@ const EntityTable = ({
       <>
         {dataTypeDisplay ? (
           isReadOnly || (dataTypeDisplay.length < 25 && !isReadOnly) ? (
-            lowerCase(dataTypeDisplay)
+            toLower(dataTypeDisplay)
           ) : (
             <Popover
               destroyTooltipOnHide
-              content={lowerCase(dataTypeDisplay)}
-              overlayInnerStyle={{ maxWidth: '420px' }}
+              content={toLower(dataTypeDisplay)}
+              overlayInnerStyle={{ maxWidth: '480px', wordBreak: 'break-word' }}
```

Comment:
why we need wordBreak? have you checked with long string? its not looking good

Reply:
<img width="695" alt="image" src="https://user-images.githubusercontent.com/71748675/202458042-f8c3594a-97ac-4635-93d0-902a1abe8a28.png">

Reply:
`raw_product_catalog` from the sample data

Reply:
@ShaileshParmar11  Updated!